notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Same here, "CRUD" isn't the most relevant term for data scientists.</p><p> </p><p>Suggestion:</p><p> </p><h2>Operate on struct data</h2><p> </p><p>Similar to pandas, BigQuery DataFrames provides a <code><a href="https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.operations.structs.StructAccessor" target="_blank">StructAccessor</a></code> . Use the methods provided in this accessor to manipulate struct data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Fixed. Thanks</p><p> </p><p> </p><p> </p><p> </p>